javaslang/src/test/java/javaslang/collection/euler/Euler17Test.java

```diff
@@ -99,7 +99,7 @@ default int letterCount(Seq<Integer> range) {
             1_000, "thousand",
             1_000_000, "million",
             1_000_000_000, "billion"
-    );
+    ).grouped(2).toSortedMap(pair -> Tuple.of((Integer) pair.get(0), (String) pair.get(1)));
```

Comment:
@ruslansennov, could you please react to my objections that you reject?
If my comments bother you and you don't find them helpful, I don't mind, but please let me know.